service-deployment/src/grafana/dockerfile

```diff
@@ -0,0 +1,51 @@
+# Copyright (c) Microsoft Corporation
+# All rights reserved.
+#
+# MIT License
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated
+# documentation files (the "Software"), to deal in the Software without restriction, including without limitation
+# the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and
+# to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+# The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED *AS IS*, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING
+# BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+# NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+# DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+
+FROM debian:jessie
+
+ENV \
+  GRAFANA_VERSION=4.6.3 \
+  GF_PLUGIN_DIR=/grafana-plugins \
+  GF_PATHS_LOGS=/var/log/grafana \
+  GF_PATHS_DATA=/var/lib/grafana \
+  UPGRADEALL=true
+
+
+COPY ./run.sh ./grafana_config.sh ./start_server.sh /usr/local/
```

Comment:
>run.sh ./grafana_config.sh ./start_server.sh [](start = 7, length = 44)

we might need to chmod +x for this shell files.

Reply:
I did this at line 42.